src/main/scala/cromwell/Main.scala

```diff
@@ -333,18 +341,25 @@ class Main private[cromwell](enableTermination: Boolean, managerSystem: () => Wo
     a <- Actions.values find (_.toString == argCapitalized)
   } yield a
 
-  private[this] def loadWdl(path: String)(f: WdlNamespace => Int): Int = {
-    Try(WdlNamespace.load(new JFile(path), WorkflowManagerActor.BackendType)) match {
-      case Success(namespace) => f(namespace)
-      case Failure(t) =>
-        println(t.getMessage)
-        exit(1)
+  private[this] def loadWdl(args: Seq[String])(f: WdlNamespace => Int): Int = {
+    /*
+      As of this writing all functions which call this have >= 2 args and are such that args(0) is the path
+      and args(1) is the backend type. Those functions will be going away soon so it's unlikely that something
+      runs afoul of this 
```

Comment:
I hear a lot of "the world is changing very soon"... what do you mean by that?  What's changing?  We probably can't make a release with a change like this unless we make it more coherent, is it an immediate ticket coming up in the backlog?  I was hoping to make a Cromwell release once Call Caching was merged... is it going to have to wait?